### ISSUE TITLE
Review all tips for relevance to deep learning and biology

### ISSUE THREAD

**Issue by agitter** [Benjamin-Lee/deep-rules#215]
To keep the scope of this article focused, the tips should be about deep learning in biology.  Several tips only pertain to one or the other.  I recommend we add more biology examples to the tips that are mostly about deep learning to keep the article relevant.

I'm less sure about what to do for the tips that describe best practices for machine learning or statistics but are not specific to deep learning.  For example, the first paragraphs of [Tip 4](https://benjamin-lee.github.io/deep-rules/#know-your-problem) are all essential but not deep learning-specific.

**Comment by Benjamin-Lee** [comment]
> I recommend we add more biology examples to the tips that are mostly about deep learning to keep the article relevant.

I agree 1000%. I'm not sure if I've mentioned it elsewhere, but examples/anecdotes are critical for cementing concepts. When I CS50, they introduced the idea of an integer overflow early on ([in lecture 1!](https://youtu.be/e9Eds2Rc_x8?t=6228)), which is kind of abstract. However, they make it very concrete and show why you really have to pay attention with examples such as the [Patriot missile battery failure](https://en.wikipedia.org/wiki/MIM-104_Patriot#Failure_at_Dhahran) which killed 28 people and the [787 overflow bug](https://arstechnica.com/information-technology/2015/05/boeing-787-dreamliners-contain-a-potentially-catastrophic-software-bug/). 

For each tip, let's ensure we have at least one biology-related example. I took a look and here's what I saw:

1. >Suppose that there are robust heritability estimates for a phenotype that suggest that the genetic contribution is modest but a deep learning model predicts the phenotype with very high accuracy. The model may be capturing signal unrelated to genetic mechanisms underlying the phenotype. In this case, a possible explanation is that people with similar genetic markers may have shared exposures.
2. >As an illustration, Rajkomar et al. [13] found that simpler baseline models achieved performance comparable with that of DL in a number of clinical prediction tasks using electronic health records, which may be a surprise to many.
3.  
4. Lots of biological applications but no examples.
5. 
6. 
7.
8.
9. >A poignant example of this lesson is...
10.

Does someone want to take the lead on figuring out some examples?
